test1.js

```diff
@@ -3,3 +3,10 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+
+var baaaar = "";
+
+if(baaaar === "bar") {
```

Comment:
Codacy has a fix for the issue: [Insert `·`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5800)
```suggestion
if (baaaar === "bar") {
```